src/info/manifest.c

```diff
@@ -489,9 +1236,14 @@ manifestLoadCallback(void *callbackData, const String *section, const String *ke
 
             file.sizeRepo = varUInt64(kvGetDefault(fileKv, MANIFEST_KEY_SIZE_REPO_VAR, VARUINT64(file.size)));
```

Comment:
I'm a little confused about the repoSize:
1) repo-size is not always present, for example:
pg_data/backup_label={"checksum":"0aaa9318dc6a0d461f28f5e649c8f9e18a623220","master":true,"size":229,"timestamp":1574780855}

2) repo-size is not the same as "size" so why defaulting it to size here?

Reply:
If it is not stored in the manifest that means it equals size -- i.e. an uncompressed backup.